executor/aggfuncs/aggfunc_test.go

```diff
@@ -508,11 +514,13 @@ func (s *testSuite) testMultiArgsMergePartialResult(c *C, p multiArgsAggTest) {
 	iter.Begin()
 	iter.Next()
 	for row := iter.Next(); row != iter.End(); row = iter.Next() {
-		partialFunc.UpdatePartialResult(s.ctx, []chunk.Row{row}, partialResult)
+		// to fix: cannot check error
```

Comment:
@lzmhhh123 
Please help to take a look at these `to fix` in this PR.

Reply:
We can leave them here. And fix them in the next PR.

Reply:
OK.
```suggestion
		// FIXME: cannot check error
```